peachjam/resources.py

```diff
@@ -72,7 +79,6 @@ def before_import_row(self, row, **kwargs):
         row["frbr_uri_date"] = frbr_uri.date
 
         if frbr_uri.actor:
-            Author.objects.update_or_create(code=frbr_uri.actor)

```

Comment:
@longhotsummer  I have removed this because creating the author with only the code without the name results in unpredictable errors since we are using the case insensitive lookup. This means that we have to ensure that the authors already exist on peachjam to avoid errors. I am open to suggestions on how to handle this.

Reply:
that's fine, as long as we get a reasonable error telling us what is missing so that we can create it.